aquadoggo/Cargo.toml

```diff
@@ -15,22 +15,27 @@ edition = "2018"
 
 [dependencies]
 anyhow = "1.0.43"
+arrayvec = "0.5.2"
 async-trait = "0.1.53"
-async-graphql = "3.0.35"
+async-graphql = "3.0.38"
 async-graphql-axum = "3.0.35"
 axum = "0.5.1"
 bamboo-rs-core-ed25519-yasmf = "0.1.1"
+base64 = "0.13"
 crossbeam-queue = "0.3.5"
 directories = "3.0.2"
 envy = "0.4.2"
-futures = "0.3.17"
+graphql_client = "0.10"
 hex = "0.4.3"
 jsonrpc-v2 = { version = "0.10.1", features = [
     "easy-errors",
     "bytes-v05",
 ], default-features = false }
 lipmaa-link = "0.2.2"
 log = "0.4.14"
+lru = "0.7.5"
+mockall = "0.11.0"
```

Comment:
`mockall` shows up in both development and application dependencies, can it be removed in one of them?